algopy_testing/tests/primitives/test_bytes.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from algopy._constants import MAX_BYTES_SIZE
+from algopy.primitives.bytes import Bytes
+
+from tests.primitives.conftest import AVMInvoker
+
+
+@pytest.mark.parametrize(
+    "value",
+    [
+        b"",
+        b"1",
+        b"0" * MAX_BYTES_SIZE,
+    ],
+)
+def test_bytes_bool(value: bytes) -> None:
+    # just test the implementation as there is no AVM equivalent
+    assert bool(Bytes(value)) == bool(value)
+
+
+@pytest.mark.parametrize(
+    ("a", "b"),
+    [
+        (b"", b""),
+        (b"1", b""),
+        (b"", b"1"),
+        (b"1", b"1"),
+        # throws "application args total length too long, max len 2048 bytes" error
+        # (b"", b"0" * MAX_BYTES_SIZE),
```

Comment:
As mentioned we should be able to use `op.bzero` to pad arguments so we can reach sizes > 2048, the python equivalent to `op.bzero(some_length)` would be 
`b"\x00" * some_length`

Reply:
It has been implemented in [this commit](https://github.com/algorandfoundation/puya/pull/212/commits/84c9cf38e63f65041ad12d18afa5ab5edf61fc06) as discussed.